docs/release-notes/8.8.asciidoc

```diff
@@ -10,6 +10,46 @@
 ==== Known issues
 
 * Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+* A UI bug can affect the Alerts table's height and make it difficult to view alerts. View the alerts in Timeline or choose one of the following workarounds to readjust the table:
+
+** Use the Alerts table **Fields** browser to remove fields until the table width is smaller than its container.
+** Adjust your OS’ zoom settings and refresh the page.
+** Zoom your browser in and then out.
```

Comment:
This should be something like 'zoom your browser in or out, and then refresh the page, issue will happen at some zoom levels and not at others.'

Reply:
```suggestion
** Zoom your browser in and out, then refresh the page. (The row height issue happens at some zoom levels.)
```